companies/services/outlook.py

```
@@ -0,0 +1,353 @@
+                    }
+                except json.JSONDecodeError:
+                    logger.warning(f"JSON 파싱 실패: {json_text[:100]}")
+
+            # JSON 파싱 실패 시 텍스트 그대로 반환
+            return {
+                "analysis": text[:300] if text else "분석 생성 실패",
+                "upside_potential": "low",
+                "signal": "sell",
+            }
+
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)